README.md

```diff
@@ -57,6 +57,12 @@ We have also covered the following topics in some detail as they might play a bi
 
 For information about Integrated Windows Authentication (IWA), see [Azure-Samples/active-directory-dotnet-iwa-v2](https://github.com/Azure-Samples/active-directory-dotnet-iwa-v2).
 
+### Chapter 3: Configure web app with SAML and OpenIDConnect
+
+|                                                  |                               |
+|--------------------------------------------------|-------------------------------|
+| [WebApp_SAML_OIDC](SAML-OIDC/WebApp_SAML_OIDC/README.md) | Signing in users with SAML and OpenIDConnect in the same web application. |
```

Comment:
```suggestion
| [WebApp_SAML_OIDC](SAML-OIDC/WebApp_SAML_OIDC/README.md) | Signing in users with SAML and OpenIDConnect in the same web application. |
```

The folder **SAML-OIDC** should be numbered like others i.e. **3-SAML-OIDC**

Reply:
@kalyankrishna1, How do we want the naming here? Is **3-SAML-OIDC** fine?